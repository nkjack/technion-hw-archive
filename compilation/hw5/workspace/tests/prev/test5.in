void maxTestsDefineThenAssign() {
	int x;
	x = 5;
	printi(x);
	print("\n");
	x = 7;
	printi(x);
}

void main() {
     maxTestsDefineThenAssign();
     print("\nI hate this course\n");
}
