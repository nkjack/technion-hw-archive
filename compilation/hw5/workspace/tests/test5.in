void foo(int a)
@pre(a > 0 and a < 15)
{
	print("OK\n");
}
void maxTestsDefineThenAssign() {
	int x;
	x = 5;
	printi(x);
	print("\n");
	x = 7;
	printi(x);
}

void main() {
	 foo(1);
     maxTestsDefineThenAssign();
     print("\nI hate this course\n");
}
