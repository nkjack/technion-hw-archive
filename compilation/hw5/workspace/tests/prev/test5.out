Loaded: ./exceptions.s
5
7
I hate this course
